code/game/objects/items/twohanded.dm

```diff
@@ -500,6 +500,13 @@
 	actions_types = list(/datum/action/item_action/startchainsaw)
 	var/on = FALSE
 
+/obj/item/twohanded/required/chainsaw/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] begins to tear [user.p_their()] head off with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	var/obj/item/bodypart/head/myhead = user.get_bodypart("head")
+	if(myhead)
+		myhead.dismember()
```

Comment:
YESH! I can tell my borging subjects to do this for EASY BRAINING!!

Reply:
emergent as fuck but i'm about to put a wrench in your plans buddy

Reply:
It's a suicide tho

Reply:
i just mean that if it's not on, it doesn't dismember. IT'LL STILL KILL!

Reply:
airtank 2.0